catalog/spatial/csw/spatial-csw-source-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/CswFilterDelegate.java

```diff
@@ -144,7 +144,7 @@ public FilterType propertyIsEqualTo(String propertyName, Date literal) {
         if (isPropertyQueryable(propertyName)) {
             return cswFilterFactory.buildPropertyIsEqualToFilter(propertyName,
                     convertDateToIso8601Format(literal),
-                    false);
+                    true);
```

Comment:
I'm instead going to make new methods that don't take these in. I think it's confusing to even pass in a boolean here if it's going to be ignored for these data, boolean, and numerical queries